content-src/asrouter/templates/SendToDeviceSnippet/SendToDeviceSnippet.jsx

```diff
@@ -27,11 +27,31 @@ function processFormData(input, message) {
   return {formData, url};
 }
 
-export const SendToDeviceSnippet = props => (
-  <SubmitFormSnippet {...props}
+function addDefaultValues(props) {
+  return {
+    ...props,
+    content: {
+      scene1_button_label: "Learn More",
+      scene2_dismiss_button_text: "Dismiss",
+      scene2_button_label: "Send",
+      scene2_input_placeholder: "YOUR EMAIL HERE",
+      locale: "en-US",
+      country: "us",
+      message_id_email: "",
+      ...props.content,
+      // Endpoint can send a reply that can start with an uppercase letter
+      include_sms: (props.content.include_sms || "false").toLowerCase(),
```

Comment:
I doubled checked and the service correctly returns boolean `true` or `false`. Apologies for asking for this change, it seems that this [line](https://github.com/mozmeao/snippets/blob/master/activity-stream/send-to-device.html#L385) confused me.